docs/mkdocs.yml

```diff
@@ -159,11 +160,11 @@ plugins:
       include_dir: 'overrides'
       module_name: 'main'
   # Configure multiple language support
-  - i18n:
-      default_language: en
-      languages:
-        en:
-          name: English
+  #- i18n:
+  #    default_language: en
+  #    languages:
+  #      en:
+  #        name: English
```

Comment:
Why are these lines being commented out?

Reply:
I could not get i18n to install on the raspberry pi, and so my local mkdocs server wouldn't start successfully. I actually restored the lines but that change apparently got munched in my rebase attempt